### PR TITLE
Stabilize and document the angle picker component; Rename to AnglePickerControl;

### PR DESCRIPTION
**Pull request by jorgefilipecosta**
## Description
This PR stabilizes and documents the angle picker component.

## How has this been tested?
I verified in the browser console that the angle picker component was now exposed as wp.components.AnglePicker.

### CODE REVIEW

**Review by chrisvanpatten — COMMENTED** [review]
I reviewed the docs and they look good to me, although I'm curious why this is `AnglePicker` vs. `AnglePickerControl`?

To me, appending "Control" is an implicit commitment that the component is wrapped in a `BaseControl`, which is the case here.

**Comment by jorgefilipecosta** [comment]
Hi @chrisvanpatten, I renamed the component to AnglePickerControl.

**Review by chrisvanpatten — APPROVED** [review]
Looks great to me!

File: packages/components/src/angle-picker-control/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Label to use for the angle picker. If not set the a translated label "Angle" is used.
+
+- Type: `String`
+- Required: No
+
+### value
+The current value of the input.
+
+- Type: `String | Number`
```
Comment (youknowriad) [inline packages/components/src/angle-picker-control/README.md]: Why is it a string or number? It's not clear what the expected value should be? 0 to 360?

Comment (jorgefilipecosta) [reply packages/components/src/angle-picker-control/README.md]: Hi @youknowriad, I documented as a String or Number because passing a value as a string or number would work well, also if the value is not between 0 and 360 things would also work well as mathematically these values are possible (although they could be represented as something between 0 and 360). But on second thought even though things work it does not mean they should be used that way so I updated type to be a number and said value should be between 0 and 360.

File: package-lock.json [hunk 2]
```diff
@@ -19511,7 +19511,7 @@
 				},
 				"node-pre-gyp": {
 					"version": "0.12.0",
-					"resolved": "https://registry.npmjs.org/node-pre-gyp/-/node-pre-gyp-0.12.0.tgz",
+					"resolved": false,
```
Comment (aduth) [inline package-lock.json]: I don't see why these changes would be expected here?

When I run `npm install` on `master` using the recommended (latest) version of NPM (6.13.7), I see this being flipped back to the value it was previously.

Comment (aduth) [reply package-lock.json]: See a59a5d9ab761bb2e0ee37eb67aab96d7f3dcf91b
